index.js

```diff
@@ -19,16 +18,7 @@ function sizeof(object) {
       else {
         var bytes = 0;
         _.forOwn(object, function (value, key) {
-          bytes += sizeof(key);
-          try {
-            bytes += sizeof(value);
-          } catch (ex) {
-            if(ex instanceof RangeError) {
-              // circular reference detected, final result might be incorrect
```

Comment:
https://github.com/google/traceur-compiler/tree/master/src/runtime/polyfills

See `Set` implementation. May also look for Babel one.